Formula/ghex.rb

```diff
@@ -16,59 +15,29 @@ class Ghex < Formula
     sha256 x86_64_linux:   "162e20b386fe920b63142876b0e0100a471d69b9737c516a9c30ed04b27d5801"
   end
 
+  depends_on "desktop-file-utils" => :build
```

Comment:
Shouldn't be needed but `gnome.post_install` forces us to have commands available when configuring via `meson`